### PR TITLE
Add Semantic Kernel Worshop

### PR DESCRIPTION
**Pull request by Philess**
Semantic Kernel Workshop in the form of an open-hack with challenges to unlock
- Azure OpenAI Service: build a chatbot trained with you own data
- Discover Semantic Kernel with the samples
- Build from scratch: semantic and native plugins
- Orchestrate Plugins
- Extra Challenge to go further

### CODE REVIEW

File: workshops/semantic-kernel-ws/workshop.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
+---
+published: false                        # Optional. Set to true to publish the workshop (default: false)
```
Comment (sinedied) [inline workshops/semantic-kernel-ws/workshop.md:2]: need to be true if you want it listed in the search page

File: workshops/semantic-kernel-ws/assets/banner.jpg
Comment (sinedied) [inline workshops/semantic-kernel-ws/assets/banner.jpg:1]: can be removed if you're not using it, it's a placeholder to make your own ;)

File: workshops/semantic-kernel-ws/workshop.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+description: This is a workshop to discover Semantic Kernel through the usage and the discovery of different topics.  # Required.
+level: advanced                         # Required. Can be 'beginner', 'intermediate' or 'advanced'
+authors:                                # Required. You can add as many authors as needed      
+  - Philippe Didiergeorges
+  - Vivien Londe
+  - Maxime Villeger
+contacts:                               # Required. Must match the number of authors
+  - philipped@microsoft.com
+  - vilonde@microsoft.com
+  - mavilleg@microsoft.com
```
Comment (olivMertens) [inline workshops/semantic-kernel-ws/workshop.md:15]: Normal ?

File: workshops/semantic-kernel-ws/workshop.md (hunk 1 above)
Comment (Philess) [reply workshops/semantic-kernel-ws/workshop.md]: good point :)

File: workshops/semantic-kernel-ws/workshop.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+duration_minutes: 1440                   # Required. Estimated duration in minutes
+tags: C#, GenAI, OpenAI                  # Required. Tags for filtering and searching
```
Comment (olivMertens) [inline workshops/semantic-kernel-ws/workshop.md:17]: Maybe indicate Python possible ?

**Review by olivMertens — APPROVED** [review]
Seems good for launch ! enjoy !

**Review by olivMertens — APPROVED** [review]
Python :)

**Review by sinedied — APPROVED** [review]

**Pull request merged**